.github/workflows/linting.yml

```diff
@@ -0,0 +1,18 @@
+name: Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
A bit late to the party, but why only on master and not main, development as well? For the `pull_request` I'd leave the branches filter off completely, since why would we want to limit it on pull_request to master?

Reply:
👍🏻 although I'd love to use GitHub actions for our other projects I think we'll stay with semaphore for some time 😆